safe/impact_functions/inundation/flood_polygon_population/test/test_flood_polygon_population.py

```diff
@@ -49,8 +50,19 @@ def test_run(self):
 
         function.hazard = hazard_layer
         function.exposure = exposure_layer
-        function.parameters['affected_field'] = 'FLOODPRONE'
-        function.parameters['affected_value'] = 'YES'
+
+        field1 = StringParameter()
```

Comment:
As discussed in our meeting, this should be expressed in KW metadata for the flood layer rather.